test/moonwall.config.json

```
@@ -404,6 +405,28 @@
                     }
                 ]
             }
+        },
+        {
+            "name": "chopsticks_frontier_template_upgrade",
+            "testFileDir": ["suites/rt-upgrade-chopsticks-frontier-template"],
+            "foundation": {
+                "type": "chopsticks",
+                "rtUpgradePath": "../target/release/wbuild/container-chain-template-frontier-runtime/container_chain_template_frontier_runtime.compact.compressed.wasm",
+                "launchSpec": [
+                    {
+                        "name": "db",
+                        "type": "parachain",
+                        "configPath": "./configs/frontierContainer.yml"
+                    }
+                ]
+            },
+            "connections": [
+                {
+                    "name": "DB",
```

@timbrinded did this change in #374

```suggestion
                    "name": "para",
```